platform/kindle/extensions/koreader/bin/koreader-ext.sh

```diff
@@ -93,55 +93,6 @@ install_koreader() {
     update_koreader "clean"
 }
 
-# Handle cre's settings...
```

Comment:
What even is all this? Some vestigial thing from 2011? >_>

Reply:
Yeah, reaaaaaallly old stuff ^^.